iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/LineXml.java

```diff
@@ -19,6 +20,10 @@
  */
 class LineXml extends AbstractConnectableXml<Line, LineAdder, Network> {
 
+    private static final String ACTIVE_POWER_LIMITS_1 = "activePowerLimits1";
+    private static final String ACTIVE_POWER_LIMITS_2 = "activePowerLimits2";
+    private static final String APPARENT_POWER_LIMITS_1 = "apparentPowerLimits1";
+    private static final String APPARENT_POWER_LIMITS_2 = "apparentPowerLimits2";
```

Comment:
If you follow first comment, you should then define the constant here based on the ones in `AbstractConnectableXml`